backend/backend.go

```diff
@@ -302,6 +304,8 @@ func GetBackendType(b []byte) (BackendType, error) {
 		return FileBackend, nil
 	case "TSS2 PRIVATE KEY":
 		return TPMBackend, nil
+	case "PUBLIC KEY":
+		return YubikeyBackend, nil
```

Comment:
We should probably do something more identifiable here, or restructure it so we can figure out a better way to detect keys.